services/seed/seed.go

```diff
@@ -0,0 +1,42 @@
+package seed
+
+import (
+	"errors"
+	"strconv"
+
+	"github.com/gin-gonic/gin"
+	"gorm.io/gorm"
+
+	"github.com/hngprojects/hng_boilerplate_golang_web/internal/models"
+)
+
+func GetUser(c *gin.Context, db *gorm.DB) (models.User, error) {
+
+	var userResp models.User
+
+	// Get the user_id from the URL
+	userIDStr := c.Param("user_id")
+
+	// convert the string id to integer
+	userID, err := strconv.Atoi(userIDStr)
+	if err != nil {
+		return userResp, err
+	}
+
+	userEmail := ""
+
+	if userID == 1 {
+		userEmail = "john@example.com"
+	} else if userID == 2 {
+		userEmail = "jane@example.com"
+	} else {
+		return userResp, errors.New("user id does not exist")
```

Comment:
I don't know why you are doing this here, but it is very wrong,
if this user exists in the db, why not use the actual record id to find the record? why hardcode this here?